.github/workflows/rustapi.yml

```diff
@@ -34,9 +34,11 @@ jobs:
       - name: Set Python path
         if: ${{ matrix.os == 'macos-latest' }}
         run: |
-          PYTHONPATH=$(realpath ../env/lib/python3.10/site-packages/):$PYTHONPATH
-          PYTHONPATH=$(realpath ../src/):$PYTHONPATH
-          echo "PYTHONPATH=$PYTHONPATH" >> "$GITHUB_ENV"
+          site=$(python3 -c '
+          from pathlib import Path;
+          print(Path("../.venv/lib/python3.11/site-packages/").resolve())
+          ')
+          echo "PYTHONPATH=$site:$PYTHONPATH" >> "$GITHUB_ENV"
```

Comment:
Couldn't the python wrapper itself update the `sys.path` at runtime when running inside a virtual environment?

This would only work if `cffi` is installed in a way in which it can be accessed by the wrapper (otherwise the wrapper won't start I guess?... not sure... maybe if it is the first thing you do...)

Something like 

```python
import sys, os, subprocess
if "VIRTUAL_ENV" in os.environ:
    sys.path.append( subprocess.run("/path/to/env/python3 -c etc").split() )
```

Reply:
Wait, there is no Python wrapper here, it's everything in Rust (calling Qibolab eventually, but it has to find it first).

Reply:
Btw, I put the wrong path in the variable, and it worked. So it seems is not required on GitHub MacOS runner.